ArcGISRuntimeSDKQt_CppSamples/Features/FeatureLayerGeodatabase/README.md

```diff
@@ -1,24 +1,52 @@
+
+## How to use the sample
+
+Pan and zoom around the map. View the data loaded from the geodatabase.
+
 ## How it works
 
-The sample creates an instance of GeodatabaseFeatureTable by supplying the path to the local mobile geodatabase. The FeatureLayer is then supplied with the GeodatabaseFeatureTable and added to the Map. The mobile geodatabase can either be obtained through ArcMap, with the Create Runtime Content geoprocessing tool, or through a feature service with the GeodatabaseSyncTask.
+1. Create a geodatabase using the provided local resource, `new Geodatabase(geodatabaseResourceUrl)`.
+2. Wait for geodatabase to load, `Geodatabase::doneLoading`.
+3. Get the 'Trailheads' `GeodatabaseFeatureTable` from the geodatabase, `Geodatabase::geodatabaseFeatureTable(tableName)`.
+4. Create feature layer using the table from above, `new FeatureLayer(geodatabaseFeatureTable)`.
+5. Add feature layer to `Map` with `Map::operationalLayers()::append(featureLayer)`.
 
-## Features
-- MapView
-- Map
-- Basemap
-- GeodatabaseFeatureTable
-- FeatureLayer
+## Relevant API
+
+* FeatureLayer
+* Geodatabase
+* GeodatabaseFeatureTable
 
 ## Offline Data
+
 Read more about how to set up the sample's offline data [here](http://links.esri.com/ArcGISRuntimeQtSamples).
 
 Link | Local Location
 ---------|-------|
 |[Los Angeles Vector Tile Package](https://www.arcgis.com/home/item.html?id=d9f8ce6f6ac84b90a665a861d71a5d0a)| `<userhome>`/ArcGIS/Runtime/Data/vtpk/LosAngeles.vtpk |
 |[Los Angeles Trailheads](https://www.arcgis.com/home/item.html?id=2b0f9e17105847809dfeb04e3cad69e0)| `<userhome>`/ArcGIS/Runtime/Data/geodatabase/LA_Trails.geodatabase |
+
+## About the data
+
+The sample shows trailheads in the greater Los Angeles area displayed on top of a vector tile basemap.
+
+## Additional information
+
+One of the ArcGIS Runtime data set types that can be accessed via the local storage of the device (i.e. hard drive, flash drive, micro SD card, USB stick, etc.) is a mobile geodatabase. A mobile geodatabase can be provisioned for use in an ArcGIS Runtime application by ArcMap. The following provide some helpful tips on how to create a mobile geodatabase file:
+
+In ArcMap, choose File > Share As > ArcGIS Runtime Content from the menu items to create the .geodatabase file (see the document: http://desktop.arcgis.com/en/arcmap/latest/map/working-with-arcmap/creating-arcgis-runtime-content.htm).
+
+Note: You could also use the 'Services Pattern' and access the Geodatabase class via a Feature Service served up via ArcGIS Online or ArcGIS Enterprise. Instead of using the Geodatabase class to access the .geodatabase file on disk, you would use GeodatabaseSyncTask point to a Uri instead. For more information review the document: https://developers.arcgis.com/qt/latest/cpp/guide/offline.htm.
```

Comment:
I've seen a few references to URI. We should change this to URL throughout - we may want to add this to our list of things to change. I've not marked all of these up in the PR